src/main/java/mate/academy/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
let's use just @Data annotation for DTO's

Reply:
done